.gitignore

```diff
@@ -1,6 +1,8 @@
 # Created by https://www.gitignore.io/api/flask
 # Edit at https://www.gitignore.io/?templates=flask
 
+.vagrant
```

Comment:
You must have added this after the files were already being tracked (you can see them listed below).

In that case you need to run `git rm --cached -r .vagrant` to explicitly remove them from the repo.